Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed an issue in `Keyboard` where the sub-script operator would return a `null` key control for the deprecated key `Key.IMESelected`. Now, an aliased `KeyControl`mapping to the IMESelected bit is returned for compability reasons. It is still strongly advised to not rely on this key since `IMESelected` bit isn't strictly a key and will be removed from the `Key` enumeration type in a future major revision. [ISXB-1541](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-1541).
 - Fixed InputControl picker not updating correctly when the Input Actions Window was dirty. [ISXB-1221](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-1221)
 - Fixed formatting issues on processor documentation page
+- Fixed an issue where a warning would be appear due to code referencing the obsolete com.unity.modules.vr package.
```

Comment:
There is a typo in the message. However, the warning part should have already been covered by `Fixed the compilation warnings when used with Unity 6.4 (ISX-2349).` just a few lines above in the changelog (as part of that change we also added a suppression message that made this warning go away). Provided that, maybe we should put this note in another section and say something like `Changed: Input System no longer depends on the obsolete com.unity.modules.vr package` ?